nifi-commons/nifi-data-provenance-utils/src/main/java/org/apache/nifi/provenance/upload/StandardFileResource.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.provenance.upload;
+
+public class StandardFileResource implements FileResource {
+
+    private final String path;
+    private final Long size;
+
+    public StandardFileResource(final String path, final Long size) {
+        this.path = path;
+        this.size = size;
+    }
+
+    public String getLocation() {
+        return path;
+    }
+
+    public Long getSize() {
+        return size;
+    }
+
+    @Override
+    public String toString() {
+        return "location=" + path + ", " +
+                "size=" + size;
```

Comment:
The toString() should include the fact that it's a StandardFileResource. E.g., `return "StandardFileResource[location=" + location + ", size=" + size + "]";`